### ISSUE TITLE
Fix the UI on the control.html

### ISSUE THREAD

**Issue by Frenchtoast0** [Frenchtoast0/Bible-Verse-Displayer#6]
UI should be responsive and look better.   Also rename "Save" button to "Search".  A simple light-blue and gray theme should look good.

**Comment by mritul** [comment]
Hi @Frenchtoast0 ! I would like to work on this issue.

**Comment by Frenchtoast0** [comment]
Thanks, I've assigned you

**Comment by mritul** [comment]
@Frenchtoast0 are there any images that are meant to be displayed which are broken currently ? 


![image](https://user-images.githubusercontent.com/84136875/193513719-3913a978-9ba8-457f-9727-2e16f53e8c81.png)

**Comment by Frenchtoast0** [comment]
No, the missing images are placeholders.  The box on the left will be sort of an "on-deck" display that only the controller sees and the one on the right will be a clone of the present.html window.
